get_all_manifests.sh

```
@@ -88,7 +83,11 @@ declare -A RHOAI_CCM_CHARTS=(
 )
 
 # RHOAI Component Charts
+# TODO(RHOAIENG-63708): pin to red-hat-data-services/workbenches-operator:rhoai-3.x
+# once the downstream fork carries charts/operator. Until then both platforms
+# consume the upstream chart; RHOAI builds rely on RELATED_IMAGE injection.
 declare -A RHOAI_COMPONENT_CHARTS=(
```

Shellcheck (0.11.0)

[warning] RHOAI_COMPONENT_CHARTS appears unused. Verify use (or export if used externally).

(SC2034)